lib/katello/repository_types/ostree.rb

```diff
@@ -24,6 +24,8 @@
   generic_remote_option :exclude_refs, title: N_("Exclude Refs"), type: Array, input_type: "text", delimiter: ",", default: [],
                          description: N_("A comma-separated list of tags to exclude during a sync. The wildcards *, ? are recognized. 'exclude_refs' is evaluated after 'include_refs'.")
 
+  generic_remote_option :depth, title: N_("Depth"), type: :number, input_type: "number", delimiter: "", default: 0, description: N_("An option to specify how many commits to traverse.")
```

Comment:
What do you think about:
```suggestion
  generic_remote_option :depth, title: N_("Depth"), type: :number, input_type: "number", delimiter: "", default: 0, description: N_("An option to specify how many ostree commits to traverse.")
```

In hammer it isn't obvious that the depth is for ostree repositories specifically.

Reply:
That's actually a very good point only just seeing now that this is also an issue for the include/exclude refs parameters.